src/dms/core/EdFi.DataManagementService.Core/OAuthManager.cs

```diff
@@ -0,0 +1,61 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+using Microsoft.Extensions.Logging;
+using System.Net;
+using System.Text;
+
+namespace EdFi.DataManagementService.Core;
+
+public interface IOAuthManager
```

Comment:
Suggestion for organization: Move into a folder/namespace called `OAuth` and break these 3 into their own files. Probably put one of those `<summary>` comment blocks on the interface at least.